hubclient/client_test.go

```diff
@@ -101,7 +101,7 @@ func TestAddClearHeaders(t *testing.T) {
 	assert.Equal(t, "another value", req.Header.Get("Another-Header-Key"))
 
 	// clear the user agent extra header value
-	client.ClearHeaderValue("User-Agent")
```

Comment:
How did it compile before?

Reply:
@tandr It didn't. Since its in test code (and hence worked for Code Sight) and probably no one tried to run tests